src/components/NewsModal/TextualContents/service/TextualContentService.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  GenericEntityService, GenericEntityServiceOpts
+} from '@terrestris/shogun-util/dist/service/GenericEntityService';
+
+import config from '../config';
+import TextualContent from '../model/TextualContent';
+
+class TextualContentService extends GenericEntityService<TextualContent> {
+
+  constructor(opts: GenericEntityServiceOpts = {
+    basePath: config.textualContentPath
+  }) {
+    super(opts);
+  }
+}
+
+export default TextualContentService;
```

Comment:
This should probably be ported over to the [shogun-util](https://github.com/terrestris/shogun-util).